src/test/java/spark/examples/books/Books.java

```diff
@@ -58,10 +58,9 @@ public static void main(String[] args) {
             Book book = books.get(request.params(":id"));
             if (book != null) {
                 return "Title: " + book.getTitle() + ", Author: " + book.getAuthor();
-            } else {
-                response.status(404); // 404 Not found
-                return "Book not found";
             }
+			response.status(404); // 404 Not found
```

Comment:
You are right, is more readable but is not necessary, it should be more readable using a method variable and one return statement only but is not the case, anyway this is about coding styles, maybe should not be accepted.